moped-editor/src/views/projects/projectsListView/ProjectsListViewExportConf.js

```diff
@@ -93,11 +90,20 @@ export const PROJECT_LIST_VIEW_EXPORT_CONFIG = {
   parent_project_name: {
     label: "Parent project name",
   },
+  parent_project_id: {
+    label: "Parent project ID",
+  },
   children_project_ids: {
     label: "Has subprojects",
     filter: resolveHasSubprojects,
   },
   components: {
-    label: "Components"
-  }
+    label: "Components",
+  },
+  interim_project_id: {
+    label: "Interim MPD (Access) ID",
+  },
+  project_and_child_project_council_districts: {
+    label: "Council districts",
```

Comment:
thru my latest update i noticed these were missing bc i actually toggled all the columns in testing... which also lead me to the idea that it would be helpful to have a select all button here?
![image](https://github.com/cityofaustin/atd-moped/assets/67130963/84703451-460a-43c9-b016-25ba5db59280)

Reply:
forgot to mention that I defer to Chia and Amenity on the column additions. I think that i remember @chiaberry  running into a bug with the adding a "select all" button but I'm not 100% on that.

Reply:
I did run into a bug with the select all button, it didnt actually return all of the columns but instead returned an empty object. maybe the pro version works better?